src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GroupFeedback/GroupFeedbackRow.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow, Form, Icon, IconButton, Row,
+} from '@edx/paragon';
+import { DeleteOutline } from '@edx/paragon/icons';
+import PropTypes from 'prop-types';
+import messages from '../../messages';
+
+export const GroupFeedbackRow = ({
+  value,
+  handleAnswersSelectedChange,
+  handleFeedbackChange,
+  handleDelete,
+  answers,
+  // injected
+  intl,
+}) => (
+
+  <div className="mb-4">
+    <ActionRow className="mb-2">
+      <Form.Control
+        value={value.feedback}
+        onChange={handleFeedbackChange}
+      />
+      <div className="d-flex flex-row flex-nowrap">
+        <IconButton
+          src={DeleteOutline}
+          iconAs={Icon}
+          alt={intl.formatMessage(messages.settingsDeleteIconAltText)}
+          onClick={handleDelete}
+          variant="primary"
+        />
+      </div>
+    </ActionRow>
+    <Form.CheckboxSet
+      onChange={handleAnswersSelectedChange}
+      value={value.answers}
+    >
+      <Row className="mx-0">
+        {answers.map((letter) => (
+          <Form.Checkbox
+            className="mr-4 mt-1"
+            value={letter.id}
+            checked={value.answers.indexOf(letter.id)}
+          >{letter.id}
```

Comment:
If you set isValid to true (add this: `isValid={value.answers.indexOf(letter.id)}`), it should have the green checkbox after rebase/merge.